bluesky/callbacks/zmqpub.py

```diff
@@ -0,0 +1,54 @@
+import zmq
+import zmq.asyncio
+import socket
+import os
+
+
+class Publisher:
+    """
+    A callback that publishes documents to a 0MQ forwarder device.
+
+    Parameters
+    ----------
+    RE : ``bluesky.RunEngine``
+        RunEngine to which the Publish will subscribe
+    host : string
+        name of host running forwarder_device
+        (See bluesky/examples/forwarder_device.py.)
+    port : int
+        frontend port of Forwarder Device
+        (See bluesky/examples/forwarder_device.py.)
+
+    Example
+    -------
+    >>> Publisher(RE, '127.0.0.1', 5567)
+    """
+    def __init__(self, RE, host, port):
+        self._host = host
+        self._port = int(port)
+        self._RE = RE
+        url = "tcp://%s:%d" % (host, port)
```

Comment:
Should use `self._port` here (`port` could be a `str` here but `self._port` is an int)